dev.Dockerfile

```diff
@@ -23,10 +23,18 @@ COPY --from=nodejsbuilder /go/src/github.com/lightninglabs/lightning-terminal /g
 # queries required to connect to linked containers succeed.
 ENV GODEBUG netdns=cgo
 
+# allow forcing a specific taproot-assets version through a build argument
```

Comment:
nit: we use complete sentences including proper capitalization and punctuation for all comments in all our projects. And where possible (depending on the file format and content), we try to adhere to the 80 character limit outside of Go code as well.

Reply:
fixed